src/pv-parahyba/utils/index.ts

```diff
@@ -6,9 +6,16 @@ export const getValueFromState = (path: string, state: IPVObject): any => {
   let finalValue: any = null;
   let currentState = state;
   const splittedPath = path.split('.');
```

Comment:
```suggestion
  const splittedPath = path.split(/[ \[.\]]+/);
```
Só alterando esse split, ache que ja resolveria, o js entende que para pegar a posição "a", ele pode acessar por "0" em string `["a", "b"]` , ex: `["a", "b"]['0'] === 'a'`

Reply:
`startup.data.adresses[0].city.name'.split(/[ \[.\]]+/); === ["startup", "data", "adresses", "0", "city", "name"]`

Reply:
GOD!